content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,87 @@
+
+In addition, Excalidraw was made possible thanks to awesome services such as [Vercel](https://vercel.com/) for hosting and pull request previews, [Crowdin](https://crowdin.com/project/excalidraw) for managing dozens of translation, [Code Sandbox](https://codesandbox.io/) to get started hacking on the project, [Sentry](https://sentry.io/) for error reporting, and [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) to keep our dependencies up to date.
+
+## Some of our coolest features
+
+### 🤝 Collaboration
+
+When the lockdown started, companies all around the world struggled to adapt to remote work. [idlewinn](https://github.com/idlewinn) and [petehunt](https://github.com/petehunt) [implemented live collaboration](https://github.com/excalidraw/excalidraw/pull/879) in Excalidraw to enable it to be used for interviews, brainstorms, presentations, and more. If you're curious, we [explained the inner workings in a blog post](/building-excalidraw-p2p-collaboration-feature/).
+
+### 🔒 Your data is encrypted
+
+Most of Excalidraw use cases involve drawing sensitive data, we architected our system such that our servers never see the content of your drawings through end-to-end encryption. The [backend](https://github.com/excalidraw/excalidraw-json) support for storing data was implemented by [lipis](https://github.com/lipis) and the client [support for encrypting](https://github.com/excalidraw/excalidraw/pull/642) the data before submitting was done by [vjeux](https://github.com/vjeux). Read more on how we are doing it in [end to end encryption](/end-to-end-encryption/) article.
+
+### 🇺🇳 Translations
+
+It was important for us early on to make sure that Excalidraw was translated into many languages so that it could be used all over the world. The [initial implementation](https://github.com/excalidraw/excalidraw/pull/638) of it was done by [fernandoalava](https://github.com/fernandoalava) and the automatic integration with our [Crowdin](https://crowdin.com/project/excalidraw) project was done by [lipis](https://github.com/lipis). UI support for right-to-left languages was [implemented](https://github.com/excalidraw/excalidraw/pull/1154) by [j-f1](https://github.com/j-f1) and [Ellinor Rapp](https://www.myfonts.com/newsletters/cc/200712.html) added support for glyphs from many more languages in the hand-written font. You can read more [about how we manage translations on the blog](/enabling-translations/).
```

Comment:
```suggestion
It was important for us early on to make sure that Excalidraw was translated into many languages so that it could be used all over the world. The [initial implementation](https://github.com/excalidraw/excalidraw/pull/638) was done by [fernandoalava](https://github.com/fernandoalava), while the automatic integration with our [Crowdin project](https://crowdin.com/project/excalidraw) was set up by [lipis](https://github.com/lipis). Support for right-to-left languages was [implemented](https://github.com/excalidraw/excalidraw/pull/1154) soon after by [j-f1](https://github.com/j-f1). To top it off, [Ellinor Rapp](https://www.myfonts.com/newsletters/cc/200712.html) designed new glyphs for several non-latin languages and added them to the Virgil hand-written font we're using. You can read more [about how we manage translations on the blog](/enabling-translations/).
```